deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -488,14 +518,58 @@ handle_tick(QName,
                                                        rabbit_fifo:make_purge_nodes(Stale)),
 
                               ok
-                      end
+                      end,
+
+                      %% ----------
+                      maybe_grow_qq_members(QName, LastActive)
                   catch
                       _:_ ->
                           ok
                   end
           end),
     ok.
 
+maybe_grow_qq_members(QName, LastActive) ->
+    {ok, Q} = rabbit_amqqueue:lookup(QName),
+    % Rely on LastActive and compare it with system_time, to figure out 30 sec intervall
+    case should_check_if_grow(LastActive) of
+        false ->
+            rabbit_log:info(">>>> TICK do nothing",[]);
+        true ->
+            rabbit_log:info(">>>> TICK check members",[]),
+            Running = rabbit_nodes:all_running(),
+            {ok, Members, _} = ra:members(amqqueue:get_pid(Q)),
+            MemberNodes = [Node || {_, Node} <- Members],
+            New = Running -- MemberNodes,
+            Size = case rabbit_policy:get(<<"target-group-size">>, Q) of
+                       undefined-> 0;
+                       V -> V
+                   end,
+            CurrentSize = length(MemberNodes),
+            case {CurrentSize < Size, New} of
+                {_, []} ->
+                    %% No new nodes to grow towards, regardless if QQ is below target
+                    ok;
+                {true, NewNodes} ->
+                    %% Sort/filter nodes before sending them to NewNodes, i.e perhaps
+                    %% Take node load or availability zones into account?
+                    add_members(Q, NewNodes, 5000, Size - CurrentSize);
+                {_,_} ->
+                    %%Target size already achieved
+                    ok
+            end
+    end.
+
+%% Return true every ~ 30 sec
+should_check_if_grow(LastActive) ->
```

Comment:
@kjnilsson How reliable is this LastActive data, in terms of 'always having a value'?
If we can use this value reliably, it can be used in a way similar to this to control interval!

Reply:
I assume last_active will update when for example a queue gets a new policy. But that is perhaps OK, it will 'reset' the interval check is all.